api/v1beta2/awscluster_webhook.go

```diff
@@ -243,26 +251,44 @@ func (r *AWSCluster) validateNetwork() field.ErrorList {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("additionalControlPlaneIngressRules"), r.Spec.NetworkSpec.AdditionalControlPlaneIngressRules, "CIDR blocks and security group IDs or security group roles cannot be used together"))
 		}
 	}
+
 	return allErrs
 }
 
-func (r *AWSCluster) validateControlPlaneLB() field.ErrorList {
+func (r *AWSCluster) validateControlPlaneLBs() field.ErrorList {
 	var allErrs field.ErrorList
 
-	if r.Spec.ControlPlaneLoadBalancer == nil {
-		return allErrs
+	// If the secondary is defined, check that the name is not empty and different from the primary.
+	// Also, ensure that the secondary load balancer is an NLB
+	if r.Spec.SecondaryControlPlaneLoadBalancer != nil {
```

Comment:
We might also want to check that the `Scheme` of the load balancer is different than the primary?

cc @JoelSpeed

Reply:
I was wondering if we should reject if the secondary LB wasn't using an `Internal` Scheme, but I think if we do put bounds on it that just making sure they're different is probably sufficient.

Reply:
I think having them different is an acceptable provision. I wouldn't want to restrict the primary to either public or private, as there are use cases for the primary to be either. But we definitely have the option to limit this API to 1 of each scheme.

If users need additional public load balancers, those are likely not for the control plane APIs and therefore should be served by the service controller rather than CAPI building out the infra IMO